recipes/soxr/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "soxr" %}
+{% set version = "0.1.3" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://downloads.sourceforge.net/project/{{ name }}/{{ name }}-{{ version }}-Source.tar.xz
+  sha256: b111c15fdc8c029989330ff559184198c161100a59312f5dc19ddeb9b5a15889
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - cmake
+    - {{ compiler('cxx') }}
+    - {{ compiler('c') }}
+    - make
+    - llvm-openmp  # [osx]
+    - libgomp  # [linux]
+  host:
+    - tinyxml
+
+test:
+  commands:
+    - test -f ${PREFIX}/include/soxr.h  # [not win]
+    - test -f ${PREFIX}/lib/libsoxr.so  # [linux]
+    - test -f ${PREFIX}/lib/libsoxr.dylib  # [osx]
+    - if not exist %PREFIX%\\Library\\include\\soxr.h exit 1  # [win]
+    - if not exist %PREFIX%\\Library\\lib\\soxr.lib exit 1  # [win]
+    - if not exist %PREFIX%\\Library\\bin\\soxr.dll exit 1  # [win]
+
+about:
+  home: https://sourceforge.net/projects/soxr/
+  license: LGPL-2.1-or-later
+  license_file: LICENCE
+  summary: 'The SoX Resampler library'
+  description: |
+    High quality, one-dimensional sample-rate conversion library
+
+  doc_url: https://sourceforge.net/p/soxr/wiki/Home/
+  dev_url: https://sourceforge.net/projects/soxr/
+
+extra:
+  recipe-maintainers:
+    - HosameldinMohamed
```

Comment:
I am happy to be a mantainer as well if it is ok for you!

Reply:
That would be great!
I'll add you!